.github/autoissuelabeler.yml

```diff
@@ -47,12 +47,36 @@
   title: "(?i).*mongo.*"
   description: "(?i).*mongo.*"
   notify: [loicmathieu]
-- labels: [area/openapi]
+- labels: [area/openapi, area/smallrye]
   title: "(?i).*openapi.*"
   notify: [EricWittmann,MikeEdgar,phillip-kruger]
-- labels: [area/graphql]
+- labels: [area/graphql, area/smallrye]
   title: "(?i).*graphql.*"
-  notify: [phillip-kruger,jmartisk]  
+  notify: [phillip-kruger,jmartisk]
+- labels: [area/reactive-messaging, area/smallrye]
+  title: "(?i).*reactive.messaging.*"
+  notify: [kenfinnigan,cescoffier,phillip-kruger]
```

Comment:
I think Ken just added that one in another PR, could you rebase and check? Talking about that one: https://github.com/quarkusio/quarkus/pull/11001 .